charts/rancher-csp-adapter/0.1.0+up0.1.0-rc2/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: 0.1.0-rc2
```

Comment:
This should be either empty, or `>1.22.0`

Reply:
Not sure what happened here - source chart has that set correctly - https://github.com/rancher/csp-adapter/blob/release/v2.6/charts/rancher-csp-adapter/Chart.yaml . Maybe something in the build scripts or in the tagging functionality?